### DISCUSSION TITLE
What is ESR?

### DISCUSSION THREAD

**Discussion by kanlukasz** [black7375/Firefox-UI-Fix#804] [Q&A] [answered]
Hi

I noticed ESR versions in the download section. What is ESR?

![image](https://github.com/black7375/Firefox-UI-Fix/assets/30685349/da510d38-d8dc-44d5-a041-bc34df02d161)

**Comment by KAGEYAM4** [comment, marked as answer]
ESR ( Extended Support Release ) similar to LTS Release of Softwares.

https://support.mozilla.org/en-US/kb/switch-to-firefox-extended-support-release-esr

**Comment by black7375** [comment]
There may be confusion. From the next release, I will leave a phrase that describes ESR.
